contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3439,6 +3445,10 @@ void exec_stmt_dbcc_checkident(PLtsql_stmt_dbcc *stmt)
 		{
 			UnlockRelationOid(table_oid, ShareLock);
 		}
+		if(msg.data)
+		{
+			pfree(msg.data);
```

Comment:
pfree(msg) also

Reply:
I think we can't free msg as we are only allocating memory to msg.data.